addons/web_editor/static/src/js/editor/snippets/s_social_media/options.js

```diff
@@ -1,14 +1,14 @@
 /** @odoo-module **/
 
 import fonts from 'wysiwyg.fonts';
-import {generateHTMLId} from 'web_editor.utils';
 import options from 'web_editor.snippets.options';
+import {generateHTMLId} from 'web_editor.utils';
 import {_t} from 'web.core';
 
 let dbSocialValues;
 let dbSocialValuesProm;
 
-options.registry.SocialMedia = options.Class.extend({
+export const SocialMedia = options.registry.SocialMedia = options.Class.extend({
```

Comment:
- Don't move a file and modify it at the same time.
- I really don't like this double assignment form but ok I guess...